aequilibrae/transit/lib_gtfs.py

```diff
@@ -70,9 +70,11 @@ def __init__(self, network, agency_identifier, file_path, day="", description=""
 
         links = self.project.network.links.data
         self.geo_links = gpd.GeoDataFrame(links, geometry=links.geometry, crs="EPSG:4326")
+        self.reprojected_links = self.geo_links.to_crs(3857)
```

Comment:
And carrying on with a different projection system?  Why? This is a recipe for disaster.